canonical/grafana-agent-operator
# Classic snap

## Issue
We want to switch to using the classically confined snap by default, but continue to have strict as an option. This is because the strictly confined snap has problems getting everything it needs from the system.
fixes #52
fixes #23 

## Solution
Added a new "classic_snap" config option.

## Testing Instructions
Fiddle with the "classic_snap" variable. You can use `snap info --verbose grafana-agent | grep confinement` to check the confinement of the currently installed snap.


## Upgrade Notes
Now uses the classically confined grafana-agent snap.
